sourcetools/buildj9tools.mk

```diff
@@ -104,7 +106,7 @@ JAR_TARGETS     += $$($1_TARGET)
 
 all : $$($1_TARGET)
 
-$(DEST_DIR)/$1.jar : $$($1_SOURCES) $$($1_OTHER_FILES)
+$$($1_TARGET) : $$($1_SOURCES) $$($1_OTHER_FILES)
```

Comment:
Does this fix something, or it is just refactoring?

Reply:
It just reuses the definition of `$1_TARGET` above, for consistency.